go/vt/tabletserver/query_executor.go

```diff
@@ -295,25 +295,33 @@ func (qre *QueryExecutor) execDDL() (*sqltypes.Result, error) {
 		return nil, vterrors.Errorf(vtrpcpb.Code_INVALID_ARGUMENT, "DDL is not understood")
 	}
 
-	conn, err := qre.tsv.te.txPool.LocalBegin(qre.ctx)
+	if qre.transactionID != 0 {
```

Comment:
I see at https://dev.mysql.com/doc/refman/5.7/en/implicit-commit.html that it is a MySQL restriction.  Since other places in vitess can create transactions, i.e. BeginExecute, should I instead just commit any active transaction and let the rest of the function work as it used to?